functions/mongoTest.mjs

```diff
@@ -0,0 +1,21 @@
+import { MongoClient } from 'mongodb';
```

Comment:
As the filename serves as part of the endpoint path, we should use dash-case, for example 'mongo-test'

Reply:
name changed